e2e/cypress/integration/archived_channel/archived_leave_channel_spec.js

```diff
@@ -270,6 +270,36 @@ describe('Leave an archived channel', () => {
 
         // * Private archived channels you are not a member above are not available on channel switcher
         cy.contains('#suggestionList', otherChannelName).should('not.exist');
+        cy.get('#quickSwitchModalLabel button.close').click();
+    });
+    it('MM-T1677 Archived channels are not shown as unread in channel switcher', () => {
```

Comment:
nit: Please separate test (`it`) block with another line for consistency.
```suggestion
    });

    it('MM-T1677 Archived channels are not shown as unread in channel switcher', () => {
```